calculate.py

```diff
@@ -35,18 +37,34 @@ def _get_single_value(self, table, params):
             # go from 1-based in the UI to 0 based in the table
             row = params["single_value_row"] - 1
             if row < 0:
-                return "Row number cannot be less than 1"
+                return i18n.trans(
+                    "MulticolumnOp._get_single_value.negativeRow",
```

Comment:
Could we please come up with shorter message IDs here? I feel modules' message IDs should reflect the user-facing logic, not module internals. (`MulticolumnOp` is a class we could easily nix; the error is a message we won't.)

How about:

* `badParam.single_value_row.tooSmall`
* `badParam.single_value_row.tooBig`
* `badParam.single_value_col.missing`
* `badParam.single_value_col.notANumber`
* `badData.percent_of_column_sum.sumIsZero`